src/test/java/com/razorpay/OrderClientTest.java

```diff
@@ -0,0 +1,121 @@
+package com.razorpay;
+
+import org.json.JSONObject;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class OrderClientTest extends BaseTest{
+
+    @InjectMocks
+    protected OrderClient orderClient = new OrderClient(TEST_SECRET_KEY);
+
+    private static final String ORDER_ID = "order_EKwxwAgItmmXdp";
+
+    /**
+     * Create order with basic details such as currency and amount details
+     * @throws RazorpayException
+     */
+    @Test
+    public void create() throws RazorpayException {
+        JSONObject request = new JSONObject("{amount:50000,currency:\"INR\",receipt:\"receipt#1\",notes:{key1:\"value3\",key2:\"value2\"}}");
```

Comment:
we can format it, for better readability

Reply:
there are some jsons are very long are we format them also ?

Reply:
yeah for beter readability, it should be formated